scheduleit/views/meetings/show.php

```diff
@@ -3,8 +3,11 @@
 require_once ABSPATH . 'config/session.php';
 
 $meeting = $database->getMeetingById($meeting_id, $_SESSION['user_id']);
+// list of onids that were invited to the event but have not registered
+$inviteList = $database->getNotRegistered($meeting['id']);
 
 if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+  if (isset($_POST['attendeeOnid'])) {
     $attendeeOnids = $_POST['attendeeOnid'];
     $link = $_POST['link'];
     $link = "http://web.engr.oregonstate.edu" . $link;
```

Comment:
Instead of hardcoding http://web.engr.oregonstate.edu, use `$_SERVER['HTTP_ORIGIN']`